src/utils/apiResponse.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Reponse handlers
+ *
+ */
+import httpStatus from 'http-status'
+
+/**
+ * Send success response
+ *
+ * @param data: data that response sends to client
+ * @param created: whether new data is created or modified
+ * @param status: specific http status code that we want to send
+ */
+interface SuccessResponse {
+	data: object
+	status: number
+}
+
+export const successResponse = (
+	data: string | object,
+	created?: boolean,
+	status?: number,
+): SuccessResponse => {
+	let responseData
+	if (typeof data === 'string') {
+		responseData = {message: data}
+	} else {
+		responseData = data
+	}
+
+	let responseStatus: number
+	if (status) {
```

Comment:
worth checking status in range 200..299. else throw Error('Invalid success status')